WordPress/src/main/java/org/wordpress/android/models/AccountModel.java

```diff
@@ -130,4 +143,28 @@ public void setEmail(String email) {
     public String getEmail() {
         return StringUtils.notNullStr(mEmail);
     }
+
+    public String getFirstName() {
+        return mFirstName;
```

Comment:
Should use `StringUtils.notNullStr(mFirstName)` here so that null is never returned. Likewise for `getLastName()` and `getAboutMe()`. This way nulls are never written to the db.